corral/util/log.pony

```diff
@@ -1,114 +1,26 @@
-type LogLevel is
-  ( LvlNone
-  | LvlErrr
-  | LvlWarn
-  | LvlInfo
-  | LvlFine
-  )
-
-primitive LvlNone
-  fun apply(): U32 => 0
-  fun string(): String => "-"
-
-primitive LvlErrr
-  fun apply(): U32 => 1
-  fun string(): String => "ERRR"
-
-primitive LvlWarn
-  fun apply(): U32 => 2
-  fun string(): String => "WARN"
-
-primitive LvlInfo
-  fun apply(): U32 => 3
-  fun string(): String => "INFO"
-
-primitive LvlFine
-  fun apply(): U32 => 4
-  fun string(): String => "FINE"
-
-primitive Level
-  fun apply(lvl: U64): LogLevel =>
-    match lvl
-    | 0 => LvlNone
-    | 1 => LvlErrr
-    | 2 => LvlWarn
-    | 3 => LvlInfo
-    | 4 => LvlFine
-    else
-      LvlFine
+use "logger"
+
+primitive LevelLogFormatter
```

Comment:
This is only used in `main.pony`, I left it in just in case, but I think this could be removed and let the program use the default formatter. I don't see a reason for the program to print the actual log level.

Reply:
What's your opinion on this @SeanTAllen ?

Reply:
yeah, lets kill it for now.

Reply:
Done